src/svelte-nodegui.ts

```diff
@@ -135,6 +135,10 @@ declare namespace svelteNodeGUI.JSX {
             import("./dom/react-nodegui/src").SystemTrayIconProps,
             import("./dom/react-nodegui/src").RNSystemTrayIcon
         >;
+        qtsvg: SvelteNodeGUIProps<
```

Comment:
I've checked this myself and it's fine as just "svg" after all – no conflicts with the Svelte language tooling.
```suggestion
        svg: SvelteNodeGUIProps<
```